swap_meet/decor.py

```diff
@@ -1,2 +1,20 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+
+    def __init__(self, id=None, width=0, length=0, condition=0 ):
+
+        super().__init__(id, condition)
+        
+        self.width = width
+        self.length = length
+        
```

Comment:
Space between instance functions can be super useful for readability, but try and keep them to just one empty line instead of 2!